src/elements/hx-drop-zone/index.spec.js

```diff
@@ -0,0 +1,78 @@
+import { fixture, oneEvent, expect } from '@open-wc/testing';
+
+/**
+ * <hx-drop-zone> component tests
+ *
+ * @type HXDropZoneElement
+ *
+ */
+describe('<hx-drop-zone> component tests', () => {
+    const template = '<hx-drop-zone>';
+
+    describe('instantiate element', () => {
```

Comment:
Please add a test for the public method:

* `drag`

https://github.com/HelixDesignSystem/helix-ui/blob/09e990d69207c176926877731278f751615f401f/src/elements/hx-drop-zone/index.js#L35-L41

Reply:
Changes have been done, Please review